### PR TITLE
Add button to logout user

### LINKED ISSUE #180: Add a Logout Button on Organization list page
**Is your feature request related to a problem? Please describe.**
Currently, there is no logout option on the site, except the one that logs the user out of the organization page. 
If a user wants to log out he has to manually go to the '/' route and the local storage is cleared which is not a widely used logout flow.

**Describe the solution you'd like**
Add a Logout Button on the /orgList page. On clicking it, localstorage will be cleared and the user will be redirected to login page.

### PR DESCRIPTION
**Pull request by Prathamesh010**
**What kind of change does this PR introduce?**
Add a button on the /orglist page to logout user and redirects to login page i.e /

**Issue Number:**
Fixes #180  

**Did you add tests for your changes?**
No.

**Snapshots/Videos:**

image

**Summary**
Currently, there is no logout option on the site, except the one that logs the user out of the manage organization page.
If a user wants to log out he has to manually go to the '/' route and the local storage is cleared which is not a widely used logout flow.

**Does this PR introduce a breaking change?**
No.

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes.

### CODE REVIEW

**Comment by yasharth291** [comment]
@Prathamesh010  see you have to understand the project first. It is not about who contributes more, It is about who solves greater bugs and with quality. Here in the commit, you have used the logout button and you are just changing the window URL to '/ ' but not changing clearing the cache and cookies and localstorage here which is not a good practice. I suggest you to add some code to clear the local storage before logout.

File: src/screens/OrgList/OrgList.tsx [hunk 1]
```diff
@@ -79,6 +79,12 @@ function OrgList(): JSX.Element {
             </a>
           </Row>
         </Navbar.Brand>
+        <button
+          className={styles.logoutbtn}
+          onClick={() => window.location.replace('/')}
```
Comment (yasharth291) [inline src/screens/OrgList/OrgList.tsx:84]: clear the local storage before logout

**Review by yasharth291 — CHANGES_REQUESTED** [review]

File: src/screens/OrgList/OrgList.tsx (hunk 1 above)
Comment (Prathamesh010) [reply src/screens/OrgList/OrgList.tsx:84]: I thought of doing it here, but then I saw the login component which clears the localstorage.

https://github.com/PalisadoesFoundation/talawa-admin/blob/35098b347817718df86e3939658c40cd17a289e0/src/screens/LoginPage/LoginPage.tsx#L13

Comment (Prathamesh010) [reply src/screens/OrgList/OrgList.tsx:84]: Can you please elaborate on the problem with reloading and localstorage?

Comment (Prathamesh010) [reply src/screens/OrgList/OrgList.tsx:84]: Considering the above issue, should i also make changes to this code here?
https://github.com/PalisadoesFoundation/talawa-admin/blob/35098b347817718df86e3939658c40cd17a289e0/src/components/AdminNavbar/AdminNavbar.tsx#L95-L98

**Comment by yasharth291** [comment]
yes you should

**Review by yasharth291 — APPROVED** [review]

**Review by palisadoes — APPROVED** [review]

**Pull request merged**